docs/en/latest/control-api.md

```diff
@@ -38,6 +38,9 @@ apisix:
     port: 9090
 ```
 
+The control API server does not support parameter matching by default, if you want to enable parameter matching
+you can add `router: 'radixtree_uri_with_parameter'` to the `control` section.
```

Comment:
Let's add a link to https://github.com/apache/apisix/blob/master/docs/en/latest/router-radixtree.md#5-parameter-match